packages/orbit-components/src/Card/index.tsx

```diff
@@ -99,52 +64,16 @@ const Card = ({
             onClose={onClose}
             header={header}
           />
-        </CardWrapper>
+        </div>
       )}
 
-      {children
-        ? React.Children.map(children, (item, key) => {
-            if (!item) return null;
-            const topRoundedBorder =
-              expandedSections.indexOf(key - 1) !== -1 || expandedSections.indexOf(key) !== -1;
-            const bottomRounderBorder =
-              expandedSections.indexOf(key + 1) !== -1 || expandedSections.indexOf(key) !== -1;
-
-            // This is used for the case when user wants to map sections and change their order
-            // related issue: #1005
-            // @ts-expect-error TODO
-            const index = Number(item.key) || key;
+      {!loading && children}
```

Comment:
do you imagine `loading ? null : children` rather than this?

Reply:
no no, I mean turning a loading condition into ternarny below: 

```
  {loading && (
        <div className="lm:border-x border-b">
          <Loading loading={loading} type="boxLoader" />
        </div>
      )}
```

into: 

```
  {loading ? (
        <div className="lm:border-x border-b">
          <Loading loading={loading} type="boxLoader" />
        </div>
      ) : children}
```

Reply:
and the condition `{!loading && children}` can be removed after

Reply:
ah okay. I made it like this for two reasons:
1. the ` : children` part is visually buried because of `prettier` formatting
2. this way it is much easier to extend conditions as necessary, e.g. `loading && loadingDebounce && <div className=...">`